apps/connect/src/env/token-bridge.mainnet.ts

```diff
@@ -348,6 +348,244 @@ export const ENV: Env = {
             Ethereum: 18,
           },
         },
+        Yakusolana: {
+          key: "Yakusolana",
+          symbol: "Yaku",
+          nativeChain: "solana",
+          tokenId: {
+            chain: "solana",
+            address: "AqEHVh8J2nXH9saV2ciZyYwPpqWFRfD2ffcq5Z8xxqm5",
+          },
+          icon: "https://assets.coingecko.com/coins/images/26785/standard/yaku_logo.png?1706636424",
+          coinGeckoId: "yaku",
+          decimals: {
+            default: 9,
```

Comment:
default decimals for all tokens should be 8 since it will not work for injective using token bridge protocol, right?

https://github.com/wormhole-foundation/wormhole/blob/main/cosmwasm/contracts/token-bridge/src/contract.rs#L472